webhook_to_fedora_messaging/endpoints/parser/parser.py

```diff
@@ -0,0 +1,63 @@
+
+from  flask import Request
+from werkzeug.datastructures.headers import EnvironHeaders
+import hashlib
+import hmac
+from webhook_to_fedora_messaging_messages.github.github import GithubMessageV1
+from ..util import fas_by_github
+
+
+
+def msg_parser(service_type: str, request: Request):
+    if service_type.lower() == "github":
+        return _github_parser(request)
```

Comment:
The function should raise `ValueError` if the `type` isn't handled. You can do that at the end of the function.